src/app.ts

```diff
@@ -14,12 +14,24 @@ async function start() {
 
     // This configures the FrsClient to use a local in-memory service called Tinylicious.
     // You can run Tinylicious locally using 'npx tinylicious'.
-    const localConfig: FrsConnectionConfig = {
```

Comment:
let's keep the example here using tinylicious and the insecuretokenprovider, and then have the commented section be for frs so that the code runs out-of-the-box

Reply:
also please scrub the tenant ID for FRS